src/cli/server.js

```diff
@@ -27,10 +27,14 @@ export default function startWebpack(config) {
   }
 
   if (program.production) {
-    compiler.run((err) => {
+    compiler.run((err, stats) => {
       if (err) {
         console.log(err);
       }
+      if (stats.hasErrors() || stats.hasWarnings()) {
+        console.log(stats.toJson().errors);
```

Comment:
Maybe some context here, also chalk or console error/warn (might have to test these)? like:
```
const info = stats.toJson();
console.error(chalk.red('Webpack encountered errors during compile: ', info.errors));
... same for warnings
```